backend/src/main/java/gov/cdc/usds/simplereport/db/repository/TestEventRepository.java

```diff
@@ -23,6 +23,14 @@
 
   public TestEvent findFirst1ByPatientOrderByCreatedAtDesc(Person p);
 
+  @Query(
+      value =
+          " SELECT DISTINCT ON (patient_id) * FROM {h-schema}test_event"
+              + " WHERE patient_id IN :patientIds"
+              + " ORDER BY patient_id, created_at DESC",
```

Comment:
Should this be on the `date_tested` column rather than `created_at`? What about `date_tested_backdate`?

Reply:
The problem there is that date_tested_backdate is nullable. We've created a little bit of a logical monster in the relationships between test_event and test_order and the flow of time, and I'm not sure there's a great and easy solution, but most people most of the time will be fine with ordering on created_at, I think.

Reply:
Yeah, it would seem to be _pretty_ weird to have a newer test for a person with an older `created_at` date than an older test

Reply:
We could add `COALSECE(date_tested_backdate, created_at) test_date` and then use that in the `ORDER BY`?

Reply:
☝️ I did this; it appears to work just fine and is certainly more consistent. There _is_ the odd scenario where in fact someone has a more recent test than their most recently entered backdated test, and the `COALESCE` corrects for this